skins/laika/src/components/pages/donation_form/PaymentType.vue

```diff
@@ -11,9 +11,12 @@
 						:disabled="disabledPaymentTypes.indexOf( paymentType ) > -1"
 						@change.native="setType">
 					{{ $t( paymentType ) }}
-					<div v-show=" disabledPaymentTypes.indexOf( paymentType ) > -1" class="has-text-dark-lighter has-margin-top-18">
+					<div v-show="disabledPaymentTypes.indexOf( paymentType ) > -1 && paymentType==='SUB'" class="has-text-dark-lighter has-margin-top-18">
```

Comment:
It might be better to move this logic to a component method to make these elements cleaner.

Your markup would then be something like: `v-show="isActivePaymentType('SUB')"`

Reply:
I need to check for the specific paymenttype value right there in the template because it should show 2 different messages in each case, so I could not move this to a method. But I did move it at least for the array check.